receiver/mysqlreceiver/metadata.yaml

```
@@ -175,6 +188,42 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [page_operations]
+  mysql.perf.table.io_waits:
```

```suggestion
  mysql.table.io_waits:
```

"perf" seems superfluous in the context of performance metrics.